### PR TITLE
fivePlusOne

### PR DESCRIPTION
**Pull request by Glazomer**

### CODE REVIEW

**Comment by Glazomer** [comment]
> Implementation is fine, it works 👍
> You need to add tests and implement other function (for all digits) as well.

Ok, thanks. But what means "fine"? You could do it better?
I'll try do add more functionality

**Comment by vitkarpov** [comment]
> But what means "fine"? You could do it better?

Sorry, I didn't mean it. It's a top-notch work 😉

**Comment by vitkarpov** [comment]
The only thing left is adding tests.

File: js/fivePlusOne.test.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+const {one, two, three, four, five, six, seven, eight, nine, add, subtract, multiply, divide} = require("./fivePlusOne.js");
+console.assert(one(add(five())) === 1 + 4,);
```
Comment (vitkarpov) [inline js/fivePlusOne.test.js]: "one plus five" is equal to 1 + 4? 🤔

Comment (Glazomer) [reply js/fivePlusOne.test.js]: Ye, noticed and fixed)

**Pull request merged**